core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -724,13 +709,44 @@ package object config {
       .checkValue(v => v > 0, "The max failures should be a positive value.")
       .createWithDefault(40)
 
-  private[spark] val EXECUTOR_PLUGINS =
-    ConfigBuilder("spark.executor.plugins")
-      .doc("Comma-separated list of class names for \"plugins\" implementing " +
-        "org.apache.spark.ExecutorPlugin.  Plugins have the same privileges as any task " +
-        "in a Spark executor.  They can also interfere with task execution and fail in " +
-        "unexpected ways.  So be sure to only use this for trusted plugins.")
-      .stringConf
-      .toSequence
-      .createWithDefault(Nil)
+  private[spark] val EXECUTOR_PLUGINS = ConfigBuilder("spark.executor.plugins")
+    .doc("Comma-separated list of class names for \"plugins\" implementing " +
+      "org.apache.spark.ExecutorPlugin.  Plugins have the same privileges as any task " +
+      "in a Spark executor.  They can also interfere with task execution and fail in " +
+      "unexpected ways.  So be sure to only use this for trusted plugins.")
+    .stringConf
+    .toSequence
+    .createWithDefault(Nil)
+
+  private[spark] val MASTER_REST_SERVER_ENABLED = ConfigBuilder("spark.master.rest.enabled")
```

Comment:
These could be in a `Standalone` object, but since they're few, probably ok to have them here for now.

Reply:
I'm OK in either way. I might miss some more configurations on standalone mode so we may have chance to move them in future.